backend/tests/unit/api/Controllers/Keycloak/UserControllerTest.cs

```diff
@@ -121,16 +122,16 @@ public async void UpdateUserAsync_Success()
             var service = helper.GetService<Mock<IPimsKeycloakService>>();
             var user = new Entity.User(Guid.NewGuid(), "test", "test@test.com");
             service.Setup(m => m.UpdateUserAsync(It.IsAny<Entity.User>())).Returns(Task.FromResult(user));
-            var model = mapper.Map<Model.Update.UserModel>(user);
```

Comment:
If this model is no longer required it would be good to delete it.

Reply:
Agreed